src/components/UserFeed.jsx

```diff
@@ -1,69 +1,126 @@
-//React
+import { useAuth } from "./AuthContext";
 import React, { useState, useEffect } from "react";
-import { CSSTransition } from "react-transition-group";
-
-//Bootstrap
-import { Link } from "react-router-dom";
-import { Container, Row, Col, Card, Image, ListGroup} from "react-bootstrap";
-import { StarFill, List } from "react-bootstrap-icons";
-
-//CSS
-import "../css/UserFeed.css";
-import "../css/Transitions.css";
-
-//Assets
-import chefIcon from "../assets/icons/chef.png";
-import gyoza from "../assets/gyozas.jpg";
+import RecipeList from "./RecipeList";
+import { Container, Spinner, ButtonGroup, Button, Row } from "react-bootstrap";
 
 function UserFeed() {
+  const [myUserName, setMyUserName] = useState(localStorage.getItem("currentUser"));
+  const { token, logout, isLogged } = useAuth();
+  const [myUserFollowing, setMyUserFollowing] = useState([]);
   const [recipes, setRecipes] = useState([]);
+  const [FollowingRecipes, setFollowingRecipes] = useState([]);
```

Comment:
OCD: this should start in lowercase (respect camelCase)
```suggestion
  const [followingRecipes, setFollowingRecipes] = useState([]);
```

Reply:
Don't commit through here, refactor properly :S

Reply:
I know it, for the different locations of the same variable...